.gitignore

```diff
@@ -1 +1,6 @@
-node_modules
\ No newline at end of file
+node_modules/
+package-lock.json
+models/users.js
```

Comment:
is there a need to ignore these files? these files cannot be push again if you mentioned here.

Reply:
Was unaware of this issue. Its my mistake. Will keep this in mind next time.